yateto/gemm_configuration.py

```diff
@@ -139,10 +139,18 @@ def __init__(self, arch, threshold: int = 128):
     self._threshold = threshold
 
   def _archSupported(self):
-    return self._arch.name.lower() in {'noarch', 'wsm', 'snb', 'hsw', 'skx', 'knc', 'knl'}
+    supported_set = {'noarch', 'wsm', 'snb', 'hsw', 'skx', 'knc', 'knl'}
 
-  def supported(self, m, n, k, sparseA, sparseB, transA, transB, alpha, beta):
-    return self._archSupported() and not (sparseA and sparseB) and (not transA and not transB) and alpha == 1.0 and beta in [0.0, 1.0]
+    if self._arch.name.lower() in supported_set:
+      return True
+    else:
+      if self._arch.host_name and self._arch.host_name.lower() in supported_set:
+        return True
+      else:
+        return False
+
```

Comment:
should be return self._arch.host_name and self._arch.host_name.lower() in supported_set